.gitignore

```diff
@@ -22,3 +24,19 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 replay_pid*
+
+# IDE-related files
+.idea/
+.code/
+.settings/
+.vscode/
+
+# Build Files
+.gradle/
+target/
+build/
+bin/
+
+# Ballerina related files
+Ballerina.lock
+.devcontainer.json
```

Comment:
Let's add EoF for all the new files where it's missing.

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-health.dicom/commit/3807383514d097fe372c936aa46ef84a627a58d3